### PR TITLE
User display

### PR DESCRIPTION
**Pull request by hanars**
This PR does to things related to the  staff user page:
- It makes installing the "django hijak" package only happen locally, and then only shows the corresponding "Login" buttons on the user page locally. The hijak plugin is really useful for testing because it lets you log in as any user so you can test different configurations, but its really too dangerous for us to have in production
- adds a column to the users table to indicate if a user is active. In djngo, you don't delete users who no longer get access to the app, because that messes up all sorts of foreign key relationships, you mark them as inactive and then they can't log in but the user model still exists for tracking purposes

### CODE REVIEW

**Comment by hanars** [comment]
Passing tests: https://travis-ci.org/github/broadinstitute/seqr/builds/741680096

**Comment by hanars** [comment]
passig codacy: https://app.codacy.com/gh/macarthur-lab/seqr/pullRequest?prid=6462236

File: ui/pages/Staff/components/Users.jsx [hunk 1]
```diff
@@ -29,15 +40,15 @@ const COLUMNS = [
...
+const Users = React.memo(({ users, loading, load, hijak }) =>
   <DataLoader load={load} content loading={false}>
     <DataTable
       striped
       idField="username"
       defaultSortColumn="email"
       loading={loading}
       data={users}
-      columns={COLUMNS}
+      columns={hijak ? HIJAK_COLUMNS : COLUMNS}
```
Comment (ShifaSZ) [inline ui/pages/Staff/components/Users.jsx:51]: Should it show HIJAK_COLUMNS only when the user is a hijak-authorized user?

File: settings.py [hunk 2]
```diff
@@ -220,15 +215,17 @@
     DEBUG = False
 else:
     DEBUG = True
-    # Enable CORS for local development
-    INSTALLED_APPS.append('corsheaders')
+    # Enable CORS and hijak for local development
+    INSTALLED_APPS = ['corsheaders', 'hijack'] + INSTALLED_APPS
```
Comment (ShifaSZ) [inline settings.py:219]: It reports `Forbidden (CSRF token missing or incorrect.)` in my environment. Are you sure it works in your environment?

**Review by ShifaSZ — CHANGES_REQUESTED** [review]
There are two issues or questions.

File: ui/pages/Staff/components/Users.jsx (hunk 1 above)
Comment (hanars) [reply ui/pages/Staff/components/Users.jsx:51]: this is a staff-only page, and hijak only shows during local development which is just staff users, so this is fine

File: settings.py (hunk 2 above)
Comment (hanars) [reply settings.py:219]: good catch, I messed up a couple settings. I'll push up the change

Comment (hanars) [reply settings.py]: this is fixed

**Review by ShifaSZ — APPROVED** [review]

**Pull request merged**